### ISSUE TITLE
Deploy contracts on main net without full blockchain sync

### ISSUE THREAD

**Issue by jaumevn** [ethereum/mist#3646]
- [x] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.9.3`
OS & Version: osx
Node version: `geth 1.7.3` 
Number of blocks synchronized: 99%
```
Hello, I have successfully deployed a few smart contracts on the rinkeby testnet using truffle and geth, all good. Now I'm trying to deploy the smart contracts on the main net,  but can't sync the blockchain, it always freezes on the last ~100 blocks. Is there any way to make it fully sync so I can deploy my contracts? It is necessary to have the hole blockchain updated to deploy smart contracts, isn't it?

I've tries almost all methods described here in many issues about sync fail. 

Thanks.
<!--

**Comment by Taledeyo** [comment]
You're known better than me.I have to learn a lot.
Thanks

**Comment by great999** [comment]
nobody can help you

**Comment by great999** [comment]
synchronized: 99% ! 
probably you have the whole internet downloaded 
how many TB ?

**Comment by rafaelosirs** [comment]
If you are in a pinch and have to deploy a contract you don't need the full node sync, just follow this simple steps instead (i found in a post and help me a lot):

1. Install https://metamask.io/ and
2. Navigate to https://remix.ethereum.org/

3. Click on settings -> choose your compiler (e.g. 0.4.19+commit.c4cbbb05). Note that optimization is unchecked.
4. Login to metmask and choose your network (i would suggest first testing using ropsten test network and using the buy button to get ETH from a free faucet)
5. Go back to the compile tab,
Delete the contents of the example contract,
Paste in your contract (it should auto compile).
6. Click on the run tab
If you contract have input data you have tu put that data on double quoute (").
Click on create.
7. Metamask will open, choose your gas price and click Submit to sign your transaction
Open metmask again, click on the transaction hash.
8. Once published. in etherscan go to the contract address and Click on Contract Code tab on etherscan, click Verify and Publish.
9. Set the name of the contract to the exact name as in remix, Compiler to the same exact version as in remix and set optimization to "No" if you didn't click it on remix in step 3.
You should now have a published contract, with source code on etherscan.

**Comment by ryanio** [comment]
We have released a new version (`0.11.0`) that has new syncing features - please give it a try! If you have any new problems that you can reproduce, please create a new issue with the details on how 
we can help. Thanks!
